miden/examples/merkle_store/merkle_store.masm

```diff
@@ -0,0 +1,32 @@
+begin
+    # push the root of the Partial Merkle Tree on the stack
+    push.0x82bb4d9a8e93582f1387611949703eecd8e4c74b904880f9c9c85f1fc1d7576d
+
+    # get the value `20` from the Partial Merkle Tree
+    push.0.2
+    mtree_get
```

Comment:
For this and other cases below, I'd do the following:

- Update the comment to say something like: "get value at depth=2, index=0 from the Partial Merkle Tree; this value should be 20"
- Add assertions after `mtree_get` to verify that the value retrieved was indeed 20.